examples/json-wrapper-ext/uniffi/bindings/python/JsonObject.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+class FfiConverterTypeJsonObject:
```

Comment:
I'm hoping we can find an abstraction like we have on the rust side in `UniffiCustomTypeWrapper`, so we just need to define `wrap` and `unwrap` here. I'd be fine with not blocking on that though.